### ISSUE TITLE
set up notifications

### ISSUE THREAD

**Issue by shaunagm** [glizzan/kybern#63]
Based on [Django-notifications](https://github.com/django-notifications/django-notifications).  Possibly include email notifications here, but if that's an extra step make a separate issue.

Putting this in Kybern for now. It may end up making sense to have a helper or wrapper that easily translates action objects to notifications, which we'd likely then put in Concord, but let's see if that ends up being necessary.

**Comment by shaunagm** [comment]
Need to think through what this looks like for the user. Things to think about:

- notifications at user level vs group level vs resource level
- sending immediately vs in a summary at the end of the day
- treating things where we're waiting on a user (so, when someone could close a condition) as higher priority for notification
- email vs on-site notifications

**Comment by shaunagm** [comment]
- [ ] notifications setting model & logic
- [ ] notifications item model & logic 
- [ ] create email template for individual & daily emails
- [ ] create signal which makes notification items after action creation/update
- [ ] create management command which runs daily emails
- [ ] write front end for updating notification settings & model
- [ ] any remaining tests
- [ ] deploy

**Comment by shaunagm** [comment]
Done in #88
